schain/schainwriter_test.go

```diff
@@ -91,7 +91,21 @@ func TestSChainWriter(t *testing.T) {
 					FD:     openrtb2.Int8Ptr(1),
 					TID:    "tid data",
 					PChain: "pchain data",
-					Ext:    json.RawMessage(`{` + seller1SChain + `}`),
+					SChain: &openrtb2.SupplyChain{
+						Complete: 1,
+						Ver:      "1.0",
+						Ext:      nil,
+						Nodes: []openrtb2.SupplyChainNode{
+							{
+								ASI: "directseller1.com",
+								SID: "00001",
+								RID: "BidRequest1",
+								HP:  openrtb2.Int8Ptr(1),
+								Ext: nil,
+							},
+						},
+					},
+					Ext: json.RawMessage(`{` + seller2SChain + `}`),
```

Comment:
Maybe change the description of this test from:
`Use schain for bidder in ext.prebid.schains; ensure other ext.source field values are retained`
to:
`Use schain for bidder in ext.prebid.schains; ensure other source field values are retained`

And to reduce confusion, perhaps change:
```Ext: json.RawMessage(`{` + seller2SChain + `}`),```
to:
```Ext: json.RawMessage(`{"some":"data"}`),```